### ISSUE TITLE
Reaction selector not visible on Firefox on iOS

### ISSUE THREAD

**Issue by aliceif** [syuilo/Misskey#13497]
### 💡 Summary

In Firefox on iOS on servers running 2024.3.0, when tapping the reaction button on a note, the reaction selector is not visible (it does darken the background though, so the button itself works).
I have also tried this on other servers that are on 2024.2.0, everything works there.

### 🥰 Expected Behavior

The reaction picker should be visible.

### 🤬 Actual Behavior

It's not.

### 📝 Steps to Reproduce

Go to any note
React to it

### 💻 Frontend Environment

```markdown
* Model and OS of the device(s): iPhone 15 Pro iOS 17.3.1
* Browser: Firefox 123.1 (38978)
* Server URL: mkultra.x27.one
* Misskey: 2024.3.0
```

### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service: manual installation from source
* Misskey: 2024.3.0
* Node: 20.11.1
* PostgreSQL: 16.2
* Redis: 7.2.4
* OS and Architecture: Ubuntu 22.04.2 aarch64
```

### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

**Comment by AlfredSK** [comment]
I can confirm that the reaction/emoji picker doesn't work with Firefox on Linux. Works with Chrome/Chromium.

I see two error messages in the browser console when clicking the + button of the reation picker::
`TypeError: right-hand side of 'in' should be an object, got null`
`TypeError: c.value.children[0] is undefined`

**Comment by Sinclair-Speccy** [comment]
I tested this myself when it was brought up to me on a Misskey server. I didn't encounter any error with the emoji picker despite testing on the iOS version of Firefox and the latest version of Misskey.

I was on iOS 17.3 before updating to iOS 17.3.1 to see if any difference was made

**Comment by AlfredSK** [comment]
Oh! 2024.3.1 was just released. It seems to address the issue. :-)

**Comment by aliceif** [comment]
Seems fixed on 2024.03.1
